CHANGES.md

```diff
@@ -16864,7 +16822,7 @@ ndif
  * Create a duplicate of the SSL_CTX's CERT in SSL_new instead of
    copying pointers.  The cert_st handling is changed by this in
    various ways (and thus what used to be known as ctx->default_cert
-   is now called ctx->cert, since we don't resort to s->ctx->[default_]cert
+   is now called ctx->cert, since we don't resort to `s->ctx->[default_]cert`
```

Comment:
Why did you backtick only the second code expression and not the first? Is there some system behind it? (E.g. only those code fragments which  contain wildcard-like characters such as `*[]{}`?)

Reply:
Yes, exactly. I did just what was necessary for vim-syntax to not think there was [emphasis] or other markdown markup. :)  It was easy, paging through the file looking for misplaced highlight blocks :)